### PR TITLE
Details Block Humanity Styles

### PR DESCRIPTION
**Pull request by Gentyspun**
Ref: https://github.com/amnestywebsite/humanity-theme/issues/231

Style the Core Details block to match the old custom Collapsable block

**Steps to test**:
1. Edit a post
2. Add both a Detail block and Collapsable block (Collapsable has been hidden from the block insterter so use code view to add )
3. Ensure both look the same in terms of styles both in the editor and on the front end

Example: http://bigbite.im/v/DusKN5

### CODE REVIEW

File: private/src/styles/blocks/core-blocks/_details.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.wp-block-details summary {
+  display: flex;
+  padding: spacing(half) 0;
+  border-bottom: 1px solid $color-black;
+  color: #262626;
```
Comment (jaymcp) [inline private/src/styles/blocks/core-blocks/_details.scss:5]:
```suggestion
  color: $color-grey-dark;
```

File: private/src/styles/blocks/core-blocks/_details.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  line-height: 1.12;
+  font-family: var(--font-family-secondary);
```
Comment (jaymcp) [inline private/src/styles/blocks/core-blocks/_details.scss:7]:
```suggestion
  font-family: var(--wp--preset--font-family--secondary);
```

**Review by jaymcp — DISMISSED** [review]
Thanks for the PR! Appearance looks good, but a couple of CSS property values need updating for FSE.

File: private/src/styles/blocks/core-blocks/_details.scss (hunk 1 above)
Comment (Gentyspun) [reply private/src/styles/blocks/core-blocks/_details.scss:5]: resolved

File: private/src/styles/blocks/core-blocks/_details.scss (hunk 2 above)
Comment (Gentyspun) [reply private/src/styles/blocks/core-blocks/_details.scss:7]: resolved

**Review by jaymcp — APPROVED** [review]
LGTM, thanks!

**Pull request merged**